stingray/lightcurve.py

```diff
@@ -194,6 +202,22 @@ def __init__(self, time, counts, err=None, input_counts=True,
                                               self.tstart + self.tseg]]))
         check_gtis(self.gti)
 
+    def change_mjdref(self, new_mjdref):
```

Comment:
What's the use case of this? In what situation would you ever want to change the MJDREF?